tests/energycal_test.py

```diff
@@ -69,6 +69,18 @@ def cal_spec(spec_data):
 
     return bq.Spectrum(spec_data, bin_edges_kev=TEST_EDGES_KEV)
 
+@pytest.fixture
```

Comment:
@cosama I don't think you want function to be a fixture. This is just a support method right? Also, per @markbandstra's comment regarding #78, how about including some fixed input/output (i.e. the gain should be `1` for points (1,1), (2,2)).

Reply:
Thanks for the comment. I added a test for chlist (0,1) at kevlist (1,3) now (to also get a non-zero offset) and removed the fixture.